server/src/main/java/org/elasticsearch/TransportVersion.java

```diff
@@ -182,6 +182,7 @@ private static TransportVersion registerTransportVersion(int id, String uniqueId
     public static final TransportVersion V_8_500_066 = registerTransportVersion(8_500_066, "F398ECC6-5D2A-4BD8-A9E8-1101F030DF85");
     public static final TransportVersion V_8_500_067 = registerTransportVersion(8_500_067, "a7c86604-a917-4aff-9a1b-a4d44c3dbe02");
     public static final TransportVersion V_8_500_068 = registerTransportVersion(8_500_068, "2683c8b4-5372-4a6a-bb3a-d61aa679089a");
+    public static final TransportVersion V_8_500_069 = registerTransportVersion(8_500_069, "5b804027-d8a0-421b-9970-1f53d766854b");
```

Comment:
@carlosdelest, @kderusso 

I generated the uuid using `uuidgen` in my local computer. Is the best way to generate uuid for TransportVersion?

Reply:
It's a good way to generate it 👍